### PR TITLE
WASM: Fix duplicate storage of newline locations

### PR DESCRIPTION
**Pull request by Shaikh-Ubaid**
fixes https://github.com/lfortran/lcompilers_frontend/issues/50

### CODE REVIEW

**Comment by Thirumalai-Shaktivel** [comment]
@Shaikh-Ubaid do you want to add a test for this?

File: src/bin/lfortran.cpp [hunk 1]
```diff
@@ -1444,10 +1444,7 @@ namespace wasm {
                         { \
                             LFortran::LocationManager::FileLocations fl; \
                             fl.in_filename = "input"; \
-                            std::ofstream out("input"); \
-                            out << input; \
                             lm.files.push_back(fl); \
-                            lm.init_simple(std::string(input)); \
```
Comment (Thirumalai-Shaktivel) [inline src/bin/lfortran.cpp:1450]: Do we get the correct error message?
Like, the contents of the error message are read from the file. So, please make sure we get the correct error message.
Thanks for fixing this.
We have to do the same in LPython as well

Comment (Shaikh-Ubaid) [reply src/bin/lfortran.cpp]: Yes, got it. Fixed.

>We have to do the same in LPython as well

Yes, I will take care. Please don't worry.

Comment (Thirumalai-Shaktivel) [reply src/bin/lfortran.cpp]: Cool, thanks.

**Comment by Shaikh-Ubaid** [comment]
>do you want to add a test for this? or can we test this in the CI?

Building to `wasm` is already tested at the `CI`. To test the `LFortran` wasm working, what we currently do is to see whether updating `lfortran` commit works in site preview and then merge the PR.

**Comment by Shaikh-Ubaid** [comment]
The examples at https://github.com/lfortran/lfortran/pull/1021 seem to work fine for me locally.

image
image

**Comment by Shaikh-Ubaid** [comment]
As the tests pass and the changes seem minor, I am merging this for the moment. Subsequently, I will attempt to update the commit version at `dev.lfortran.org`. Let's see if it works.

**Comment by certik** [comment]
Perfect, thanks for the quick fix!

**Pull request merged**